php_phongo.c

```diff
@@ -1638,30 +1613,16 @@ static void php_phongo_cursor_iterator_rewind(zend_object_iterator *iter TSRMLS_
 {
 	php_phongo_cursor_iterator *cursor_it = (php_phongo_cursor_iterator *)iter;
 	php_phongo_cursor_t        *cursor = cursor_it->cursor;
+	const bson_t               *doc;
 
 	php_phongo_cursor_free_current(cursor);
 	cursor_it->current = 0;
```

Comment:
Then retrieving the key would be off-by-one between calling rewind() and next(), wouldn't it?